### PR TITLE
chore(design): design workflow for PVC and SC options

### PR DESCRIPTION
**Pull request by mittachaitu**
Signed-off-by: mittachaitu <sai.chaithanya@mayadata.io>

## Pull Request template

**What this PR does?**:
This PR adds workflows for various parameters of PVC and SC

**Does this PR require any upgrade changes?**:
No

**If the changes in this PR are manually verified, list down the scenarios covered:**:

**Any additional information for your reviewer?** :
_Mention if this PR is part of any design or a continuation of previous PRs_

**Checklist:**
- [ ] Fixes #<issue number>
- [ ] PR Title follows the convention of  `<type>(<scope>): <subject>`
- [ ] Has the change log section been updated?
- [ ] Commit has unit tests
- [ ] Commit has integration tests
- [ ] (Optional) Are upgrade changes included in this PR? If not, mention the issue/PR to track:
- [ ] (Optional) If documentation changes are required, which issue on https://github.com/openebs/openebs-docs is used to track them:

### CODE REVIEW

**Comment by mittachaitu** [comment]
@pawanpraka1 @prateekpandey14 @w3aman  Can you PTAL at this PR?

File: design/lvm/resize_workflow.md [hunk 1]
```diff
@@ -131,6 +131,7 @@ Below is high level sequence diagram for volume expansion workflow
 
 ### Test Plan
 A test plan will include following test cases:
+<<<<<<< HEAD
```
Comment (kmova) [inline design/lvm/resize_workflow.md:134]: @mittachaitu -- can you check this please.

File: docs/persistentvolumeclaim.md [hunk 2]
```diff
@@ -42,7 +42,7 @@ Following matrix shows supported PersistentVolumeClaim parameters for lvm-localp
       <td rowspan=2> <a href="#volumemode-optional"> VolumeMode </a> </td>
       <td> Block </td>
       <td> Supported </td>
-      <td rowspan=2> <a href="https://github.com/openebs/lvm-localpv/blob/master/e2e-tests/apps/percona/deployers/run_e2e_test.yml"> Yes </a> <br> <i> Test cases available for Filesystem mode </i> </br> </td>
+      <td rowspan=2> <a href="https://github.com/openebs/lvm-localpv/blob/master/e2e-tests/apps/percon  a/deployers/run_e2e_test.yml"> Yes </a> <br> <i> Test cases available for Filesystem mode </i> </br> </td>
```
Comment (kmova) [inline docs/persistentvolumeclaim.md:45]: @mittachaitu -- nitpick..

File: design/lvm/resize_workflow.md (hunk 1 above)
Comment (mittachaitu) [reply design/lvm/resize_workflow.md]: Updated.. Missed while resolving comments

File: docs/persistentvolumeclaim.md (hunk 2 above)
Comment (mittachaitu) [reply docs/persistentvolumeclaim.md]: Updated with valid link

**Review by kmova — APPROVED** [review]

**Pull request merged**